test/integration/puppetlabs/trapperkeeper/services/scheduler/scheduler_service_test.clj

```diff
@@ -345,5 +345,104 @@
           (is (empty? (get-jobs service))))
         (deref job-done)))))
 
+(defn distances
+  "Calculate the distances between each item in a sequence."
+  [v]
+  (map #(- %2 %1) v (rest v)))
+
+(deftest ^:integration test-interval-after
+  (let [num-runs 10 ; let it run a few times, but not too many
+        initial-delay 500
```

Comment:
Would it be better to have an initial-delay that was different than the recurring-delay?

Reply:
Yes, that's a good idea.